File: Gombang/Weekly Contest 410/3248_Snake_in_Matrix.cs
using System.Collections.Generic;

public class Solution
{
	public int FinalPositionOfSnake(int n, IList<string> commands)
	{
		Dictionary<string, int> commandDic = new Dictionary<string, int>() {
			{ "UP", -n },
			{ "DOWN", n },
			{ "LEFT", -1 },
			{ "RIGHT", 1}
		};

		int[] array = new int[n * n];
		int result = 0;

		foreach (string command in commands)
		{
			result += commandDic[command];
		}

		return result;
	}
}

File: Seori/Weekly Contest 410/3248. Snake in Matrix.py
class Solution:
    def finalPositionOfSnake(self, n: int, commands: List[str]) -> int:
        # i와 j의 좌표를 구하고, 이동한 좌표를 반환한다.
        i, j = 0, 0
        for command in commands:
            if command == 'RIGHT':
                j += 1
            elif command == 'LEFT':
                j -= 1
            elif command == 'DOWN':
                i += 1
            elif command == 'UP':
                i -= 1

        return n * i + j